src/torManager/torManager.test.ts

```diff
@@ -98,4 +98,55 @@ describe('Tor manager', () => {
     expect(tor.torHashedPassword).toHaveLength(61)
     expect(tor.torPassword).toHaveLength(32)
   })
+
+  it('tor spawn repeating 3 times with 1 second timeout and repeating will stop after that', async () => {
+    const torPath = torBinForPlatform()
+    const [controlPort] = await fp(9051)
+    const httpTunnelPort = (await fp(controlPort as number + 1)).shift()
+    const socksPort = (await fp(httpTunnelPort as number + 1)).shift()
+    const libPath = torDirForPlatform()
+    const tor = new Tor({
+      appDataPath: tmpAppDataPath,
+      socksPort,
+      torPath: torPath,
+      controlPort,
+      httpTunnelPort,
+      options: {
+        env: {
+          LD_LIBRARY_PATH: libPath,
+          HOME: tmpAppDataPath
+        },
+        detached: true
+      }
+    })
+
+    try {
+      await tor.init(3, 1000)
+      await tor.kill()
+    } catch { console.log('to many try of tor spawn') }
+  })
+
+  it('tor is initializing correctly with 40 seconds timeout', async () => {
+    const torPath = torBinForPlatform()
+    const [controlPort] = await fp(9051)
+    const httpTunnelPort = (await fp(controlPort as number + 1)).shift()
+    const socksPort = (await fp(httpTunnelPort as number + 1)).shift()
+    const libPath = torDirForPlatform()
+    const tor = new Tor({
+      appDataPath: tmpAppDataPath,
+      socksPort,
+      torPath: torPath,
+      controlPort,
+      httpTunnelPort,
+      options: {
+        env: {
+          LD_LIBRARY_PATH: libPath,
+          HOME: tmpAppDataPath
+        },
+        detached: true
+      }
+    })
+    await tor.init(3, 40000)
+    await tor.kill()
```

Comment:
this test case check happy path correct ?
At very least, before kill tor I would check that tor is actually running. 
The way you wrote it right now, this test would pass even if `TorManager.init()` function is blank, right?